VIM_COMPARISON.org

```diff
@@ -0,0 +1,122 @@
+| h            | ~meow-left~                                  | character left                                                            |
+| j            | ~meow-next~                                  | next line                                                                 |
+| k            | ~meow-prev~                                  | previous line                                                             |
+| l            | ~meow-right~                                 | character right                                                           |
+| w            | ~meow-next-word~                             | next word                                                                 |
+| W            | ~meow-next-symbol~                           | next symbol                                                               |
+| b            | ~meow-back-word~                             | last word                                                                 |
+| B            | ~meow-back-symbol~                           | last symbol                                                               |
+| e            | ~meow-mark-word~                             | end of word                                                               |
+| E            | ~meow-mark-symbol~                           | end of symbol                                                             |
+| ge           |                                              | end of previous word                                                      |
+| gE           |                                              | end of previous symbol                                                    |
+| 0            | ~negative-argument~ - ~meow-line~            | beginning of line                                                         |
+| 0            | ~meow-line~ - ~meow-reverse~                 | beginning of line                                                         |
+| ^            | ~meow-join~                                  | first none whitespace character                                           |
+| $            | ~meow-line~                                  | EOL                                                                       |
+| f - <symbol> | ~meow-find~ - <symbol>                       | forward to next symbol                                                    |
+| F - <symbol> | ~negative argument~ - ~meow-find~ - <symbol> | backward to last symbol                                                   |
+| t - <symbol> | ~meow-till~ - <symbol>                       | till next symbol                                                          |
+| %            | ~meow-block~                                 | jump to matching paren                                                    |
+|              |                                              | not one to one replacement can jump from anywhere inside the block to end |
```

Comment:
No sure why the Vim column and Meow column are empty here. Something missing perhaps? Could you double check?

Reply:
I did that on purpose because of the long description but I could put it into a single row

Reply:
I see. To me it looks a bit confusing though. 

I might do sth like this:
1. Add a star `*` or some similar symbol as a superscript, indicating this description has extra explanation somewhere further in this doc. 
2.  Maybe right below this very table, put the extra information/annotation for the description.

What do you think? Feel free to disagree. I'm just thinking this might make the table format more consistent

Reply:
changed it now, looked really strange in github and it takes care of wrapping by itself

Reply:
Yes, it looks strange in github. @DogLooksGood and @eshrh may want to have a final quick review a little bit later. 
It looks good to me.  Thank you @snowiow ^_^